### PR TITLE
fine, I'll do it myself. - add snoo.habedieeh.re and onion version

### PR DESCRIPTION
**Pull request by shruuub**

### CODE REVIEW

File: instances.json [hunk 1]
```diff
@@ -32,9 +37,10 @@
       "version": "v0.1.0"
     },
     {
-      "i2p": "http://vernchoyi4qjf6sz5vi57sskoelw5lsf53u62i5uvktzpa6sxv3a.b32.i2p",
```
Comment (Daniel-Valentine) [inline instances.json:35]: I don't think we want to get rid of this. Did you run your update script with an I2P router running?

**Review by Daniel-Valentine — CHANGES_REQUESTED** [review]

File: instances.json (hunk 1 above)
Comment (shruuub) [reply instances.json:35]: huh, I did not. From the readme and the script's output it seemed like it would just be skipped

Comment (shruuub) [reply instances.json:35]: Okay, so I just added it via the gh web editor (hope that doesnt break things with json and stuff). Sorry for the inconvenience.
_btw the pr message wasn't meant to be mean if it seems that way, just doing the funny_

File: instances.json [hunk 2]
```diff
@@ -31,6 +36,12 @@
       "country": "CA",
       "version": "v0.1.0"
     },
+    {
+      "onion": "http://snoo.habeehrhadazsw3izbrbilqajalfyqqln54mrja3iwpqxgcuxnus7eid.onion",
+      "country": "CA",
+      "version": "v0.1.0",
+      "description": "onion of snoo.habedieeh.re"
+    }
```
Comment (Daniel-Valentine) [inline instances.json:44]:
```suggestion
    },
```

**Review by Daniel-Valentine — CHANGES_REQUESTED** [review]

File: instances.json (hunk 1 above)
Comment (Daniel-Valentine) [reply instances.json:35]: No worries. I did not take offense. I'll also edit the README to clarify that onion/i2p sites are all or nothing updates.

**Pull request merged**